docs/source/Support/bskReleaseNotes.rst

```diff
@@ -62,6 +62,8 @@ Version  |release|
   :ref:`scenarioRoboticArm` scenarios.
 - Fixed issue in which reading ``RWModel`` from RW message payloads when :ref:`vizInterface` was also
   imported would return a Swig Object instead of an enumerated integer.
+  - Fixed SWIG array handling for message payloads on macOS, particularly addressing issues with uint8_t arrays and other
```

Comment:
This line shouldn't have 2 blank spaces to begin with. It causes this bullet to be combined with the prior bullet.  If you make "-" line up with the left side it should render properly.